packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+# Tab
+
+Tabs are a navigational component used to organize content by grouping similar information on the same page. They allow content to be viewed without having to navigate away from that page or route.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
```

Comment:
Any reason to not have the rendered code match the doc code below?

Reply:
Nope. Fixed.